app/src/main/java/nl/nfi/cellscanner/MainActivity.java

```diff
@@ -41,7 +42,7 @@
 import static nl.nfi.cellscanner.recorder.RecorderUtils.inRecordingState;
 
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends AppCompatActivity implements SharedPreferences.OnSharedPreferenceChangeListener {
```

Comment:
for my understanding; dit luistert gewoon naar elke verandering in opgeslagen waardes oid?

Reply:
Jep